bigtable-hbase-1.x-parent/bigtable-hbase-1.x/pom.xml

```diff
@@ -105,5 +105,31 @@ limitations under the License.
       <version>${junit.version}</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-api</artifactId>
+      <scope>test</scope>
```

Comment:
Although this module doesn't reference grpc-api directly, it's declared as a compile time dep of bigtable-client-core. So if you override it to test here it will be missing during runtime.

Please mark this as compile

Reply:
I just wanted to clear before changing the scope. We need these dependencies for `TestBigtableConnection.java` to for the Server for FakeDataService and BigtableGrpc.

Reply:
These are direct dependencies of TestBigtableConnection which is in test scope, but they are transitive dependencies of bigtable-client-core, which needs them during runtime. 
test scope inherits compile scope, which inherits runtime scope.

So runtime scope would probably be the most correct. But I'm not sure how that will play with shading, so I'm suggesting compile scope to be safe. Maybe add a comment explaining this next to each declaration